### PR TITLE
Added tab summary for compute, storage, networking

### PR DESCRIPTION
**Pull request by infosatheesh2020**

### CODE REVIEW

**Comment by infosatheesh2020** [comment]
@tsubasaxZZZ need your help to review and merge this

File: build/template_kql/networking/networking_afd_resources_details_summary.kql [hunk 1]
```diff
@@ -0,0 +1,4 @@
+${extend_resource}
+| project Type, SkuName, Kind, SubscriptionId, Name, State, Location, ResourceGroup, Zones, AvailabilityZone, AvailabilitySet, Capacity, FaultDomain, OSDisk, ReadReplica, AutomaticFailover, MultipleWriteLocations, StorageAutogrow, ReplicationRole, Backup = tolower(Backup), Environment
+| where Type == 'microsoft.network/frontdoors' or Type == 'microsoft.cdn/profiles'
+| summarize ResourceTotal=count(Name), ClassicSKU=dcountif(Name, SkuName == "standard_microsoft"), NotActive=dcountif(Name, (State != "active" and State != "enabled"))
```
[the remark this answers is not in the document]
Comment (tsubasaxZZZ) [reply build/template_kql/networking/networking_afd_resources_details_summary.kql:4]: I've just found the following document:
https://learn.microsoft.com/en-us/azure/cdn/cdn-change-provider

File: build/template_kql/compute/compute_vm_resources_details_summary.kql [hunk 2]
```diff
@@ -0,0 +1,4 @@
+${extend_resource}
+| project Type, SkuName, Kind, SubscriptionId, Name, State, Location, ResourceGroup, Zones, AvailabilityZone, AvailabilitySet, Capacity, FaultDomain, OSDisk, ReadReplica, AutomaticFailover, MultipleWriteLocations, StorageAutogrow, ReplicationRole, Backup = tolower(Backup), Environment
+| where Type == 'microsoft.compute/virtualmachines'
+| summarize ResourceTotal=count(Name), NotRunning=dcountif(Name, State != "vm running"), NoAVZone=dcountif(Name, AvailabilityZone == "not configured"), NoAVSet=dcountif(Name, AvailabilitySet == "not configured"), OSDiskRecommendation=dcountif(Name, (OSDisk != "premium_lrs" and OSDisk != "ephemeral")), NoBackup=dcountif(Name, Backup == "unknown")
```
[the remark this answers is not in the document]
Comment (infosatheesh2020) [reply build/template_kql/compute/compute_vm_resources_details_summary.kql:4]: Thanks. I am adding additional fields for unhealthy backup as well. Backup unknown means backup not configured. Unhealthy means configured but unhealthy (say no backup scheduled due to VM shutdown, etc.)

[the remark this answers is not in the document]
Comment (tsubasaxZZZ) [reply build/template_kql/compute/compute_vm_resources_details_summary.kql:4]: Thank you. In my environment, also Unhealthy means no configured. On the contrary, be shutdown and configured machine is healthy...
image

[the remark this answers is not in the document]
Comment (infosatheesh2020) [reply build/template_kql/compute/compute_vm_resources_details_summary.kql]: okay. I was validating your environment and if the backup config is not found then it is unknown. If backup config is found, then we get either healthy or unhealthy based on the last backup. Anyway we have to cover unknown where backup is clearly not configured, and unhealthy status should be fixed by user upon more investigation.

**Comment by infosatheesh2020** [comment]
@tsubasaxZZZ Can you check now?

File: build/template_kql/networking/networking_afd_resources_details_summary.kql [hunk 3]
```diff
@@ -1,4 +1,4 @@
 ${extend_resource}
 | project Type, SkuName, Kind, SubscriptionId, Name, State, Location, ResourceGroup, Zones, AvailabilityZone, AvailabilitySet, Capacity, FaultDomain, OSDisk, ReadReplica, AutomaticFailover, MultipleWriteLocations, StorageAutogrow, ReplicationRole, Backup = tolower(Backup), Environment
 | where Type == 'microsoft.network/frontdoors' or Type == 'microsoft.cdn/profiles'
-| summarize ResourceTotal=count(Name), ClassicSKU=dcountif(Name, SkuName == "standard_microsoft"), NotActive=dcountif(Name, (State != "active" and State != "enabled"))
\ No newline at end of file
+| summarize ResourceTotal=count(Name), ClassicSKU=dcountif(Name, (SkuName == "standard_microsoft" or SkuName == "standard_akamai")), NotActive=dcountif(Name, (State != "active" and State != "enabled"))
```
Comment (tsubasaxZZZ) [inline build/template_kql/networking/networking_afd_resources_details_summary.kql:4]: Sorry again. It should add "classic_frontdoor". It looks inconsistent between summary and details. 
image
To be:
```kusto
| summarize ResourceTotal=count(Name), ClassicSKU=dcountif(Name, (SkuName == "standard_microsoft" or SkuName == "standard_akamai" or SkuName == "classic_frontdoor")), NotActive=dcountif(Name, (State != "active" and State != "enabled"))
```

**Pull request merged**